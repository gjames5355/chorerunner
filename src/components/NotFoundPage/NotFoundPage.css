.notfound-container {
  height: 100vh;
}

.notfound-container h2 {
  margin: 0;
  padding-top: 300px;
  color: blanchedalmond;
}