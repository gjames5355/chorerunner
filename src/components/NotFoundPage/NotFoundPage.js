import './NotFoundPage.css';

import React from 'react';

export default function NotFoundPage() {
  return (
    <div className='notfound-container'>
      <h2>Page not found</h2>
    </div>
  );
}
