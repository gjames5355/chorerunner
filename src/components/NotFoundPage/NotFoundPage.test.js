import React from 'react';
import ReactDOM from 'react-dom';
import NotFoundPage from './NotFoundPage';
import renderer from 'react-test-renderer';

describe('NotFoundPage component testing', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');

    ReactDOM.render(<NotFoundPage />, div);

    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders UI as expected', () => {
    const tree = renderer.create(<NotFoundPage />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
