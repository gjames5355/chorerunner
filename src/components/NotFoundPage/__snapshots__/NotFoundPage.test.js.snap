// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage component testing renders UI as expected 1`] = `
<div
  className="notfound-container"
>
  <h2>
    Page not found
  </h2>
</div>
`;
